Geometry/HcalCommonData/interface/HcalCellType.h

```diff
@@ -8,7 +8,7 @@
 #include <vector>
 #include <iostream>
 #include "DataFormats/HcalDetId/interface/HcalSubdetector.h"
-#include "CLHEP/Units/GlobalSystemOfUnits.h"
```

Comment:
@slava77 , There is a strong recommendation to use in headers  

#include <CLHEP/Units/SystemOfUnits.h>. 

Would it be possible to fix? In the source it is not required.

Reply:
> @slava77 , There is a strong recommendation to use in headers
> 
> #include <CLHEP/Units/SystemOfUnits.h>.
> 
> Would it be possible to fix? In the source it is not required.

good to know, that's effectively what I did in this PR (`git grep GlobalSystemOfUnits | grep h:` to catch the header files and then replaced to `SystemOfUnits`).